test/test.js

```diff
@@ -252,54 +251,39 @@ describe( 'Core Methods: ', function() {
 		} )
 
 		it( 'return correctly formatted msg', function() {
-			app.state.severity = 'Warning'
-			app.cache.file = 'testReporter'
-			app.cache.col = 0
-			app.cache.lineNo = 1
-			app.cache.origLine = 'Reporter Lyfe*'
 			app.cache.rule = 'universal'
-			var msg = 'universal disallowed'
 			var expectedOutput = '\u001b[4mtestReporter\u001b[24m\n1:0 \u001b[90muniversal\u001b[39m \u001b[33mwarning\u001b[39m universal disallowed'
```

Comment:
should just use `chalk.stripColor`, but whatever. Can fix when we refactor tests